torch/_export/utils.py

```diff
@@ -144,6 +147,10 @@ def get_keystr(key_path: KeyPath) -> str:
                                 )
                 else:
                     if arg_dim != node_dim:
+                        if disable_forced_specializations and isinstance(
```

Comment:
So this is the case where multiple symbols occur, yes? Do we need a flag here? Maybe it's better to always ignore, or always try harder and always convert them to checks.

Reply:
Yep that's correct. Making a note here about maybe using these as guards and substituting in concrete values.